WPThemeReview/Sniffs/PluginTerritory/ForbiddenFunctionsSniff.php

```diff
@@ -44,6 +45,16 @@ public function getGroups() {
 					'add_shortcode',
 					'register_taxonomy_for_object_type',
 					'flush_rewrite_rules',
+					'register_block_type',
```

Comment:
Suggestion: Maybe this should be a separate group (within the same sniff) ?

Reply:
Yes, this makes sense. I'll place it in `editor-blocks` group :+1: